jsk_tools/src/sanity_diagnostics.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+# https://github.com/start-jsk/jsk_apc/blob/master/jsk_arc2017_baxter/node_scripts/sanity_check_for_pick.py # NOQA
+
+from diagnostic_msgs.msg import DiagnosticStatus
+import diagnostic_updater
+from jsk_tools.sanity_lib import checkNodeState, checkTopicIsPublished
+import os
+import rospy
+import sys
+import yaml
+
+
+class SanityDiagnostics(object):
+    """
+    This node publishes essential robot topic and node status to /diagnostics.
+
+    The yaml file is like the following:
+    topics:
+      - /kinect_head/rgb/image_raw
+      - /tf
+    nodes:
+      - /kinect_head/kinect_head_nodelet_manager
+      - /respeaker_node
+    """
+    def __init__(self):
+        yaml_path = rospy.get_param(
+            '~sanity_targets', "/var/lib/robot/sanity_targets.yaml")
+        yaml_path = '/home/naoya/sanity_targets.yaml'
+        duration = rospy.get_param('~duration', 60)
+        if os.path.exists(yaml_path):
+            # Get sanity target topic and node names
+            with open(yaml_path) as yaml_f:
+                sanity_targets = yaml.load(yaml_f)
```

Comment:
`yaml.load(yaml_f, Loader=yaml.SafeLoader)` is better for `python3`.

Reply:
Thank you.

I use `yaml.SafeLoader`
[`dcce522` (#1727)](https://github.com/jsk-ros-pkg/jsk_common/pull/1727/commits/dcce522f624da5b1b6df2e154b9c5af53cc55bd8)